index.bs

```diff
@@ -4027,6 +4131,52 @@ This operation is ignored if it is invoked in an [=authenticator session=] which
 or [=authenticatorGetAssertion=] operation currently in progress.
 
 
+### The <dfn>silentCredentialDiscovery</dfn> operation ### {#sctn-op-silent-discovery}
+
+This is an OPTIONAL operation authenticators MAY support to enable {{CredentialMediationRequirement/conditional}}
+[=user mediation=].
+
+It takes the following input parameter:
+
+: |rpId|
+:: The caller's [=RP ID=], as <a href='#GetAssn-DetermineRpId'>determined</a> by the user agent and the client.
+
+If [=silentCredentialDiscovery=] is not supported, the operation always returns an empty [=list=].
```

Comment:
Well, I think this is there to handle the case where the client requests silentCredentialDiscovery even though the authenticator does not support it. 

A well behaved client using an authenticator API that is able to indicate whether the authnr supports the operation ought not do that. 

I think the intention is to not throw an user-visible error.

Reply:
> If the authenticator doesn't support the operation at all, wouldn't it be the client that performs the fallback?

Yeah and we're already prescribing this on the [[DiscoverFromExternalSource]] algorithm.

> A well behaved client using an authenticator API that is able to indicate whether the authnr supports the operation ought not do that.

Yeah, and there is no scenario I can think of where this would malfunction.

Removed.